management/templates/management/cohort_list.html

```diff
@@ -39,6 +39,17 @@
                     <th data-filter="false" data-sorter="false">Action</th>
                 </tr>
             </thead>
+            <tfoot>
+                <tr>
+                    <th>Code</th>
+                    <th>Description</th>
+                    <th>Group</th>
+                    <th data-filter="false" data-sorter="false">Action</th>
+                </tr>
+                <tr>
+                    {% include 'seumich/ts_pager.html' with colspan="6" %}
```

Comment:
Nitpick: I'm fine with this. Probably in an ideal world we'd refactor so that `ts_pager.html` was somewhere that both `seumich` and `management` could find, rather than pulling from `seumich` here. May not take much to change (though I'm not sure where the template should be exactly), but it also doesn't seem that important right now. I can create an issue if you think it's worth considering.

Reply:
Yeah I didn't know where to put it in the middle either.